ngraph/test/serialize.cpp

```diff
@@ -197,26 +182,6 @@ TEST(serialize, constant)
     EXPECT_TRUE(found);
 }
 
-TEST(benchmark, serialize)
```

Comment:
Same here - why are you removing this test?

Reply:
Model contains Add v0 operation